{"uuid": "e87301ef-b4d3-4328-82e5-b4be4de291b3", "children": ["5d23db7a-d647-4a8d-b552-9af9450cfac9", "ade182e9-0bf4-4244-98cf-1c988f8fde4c", "4fd64266-9b83-4fd9-ab91-204bca552ac7", "73a9c347-b42e-456d-ba96-536fbd76b0c1", "d5ff9b5f-2777-4d10-9dca-fe857284fb1b"], "befores": [{"name": "get_token", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\python_code\\\\pytest_demo/test_case/inter_case/extract.yaml'\n", "trace": "  File \"E:\\programs\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\programs\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\programs\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"E:\\python_code\\pytest_demo\\test_case\\inter_case\\conftest.py\", line 19, in get_token\n    return yam.read_inter_yaml('extract.yaml')['accessToken']\n  File \"E:\\python_code\\pytest_demo\\common\\yaml_util.py\", line 31, in read_inter_yaml\n    with open(self.root_path + r'/test_case/inter_case/' + file_name,mode='r',encoding='utf-8') as f:\n"}, "start": 1652772923521, "stop": 1652772923521}, {"name": "get_token", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\python_code\\\\pytest_demo/test_case/inter_case/extract.yaml'\n", "trace": "  File \"E:\\programs\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\programs\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\programs\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"E:\\python_code\\pytest_demo\\test_case\\inter_case\\conftest.py\", line 19, in get_token\n    return yam.read_inter_yaml('extract.yaml')['accessToken']\n  File \"E:\\python_code\\pytest_demo\\common\\yaml_util.py\", line 31, in read_inter_yaml\n    with open(self.root_path + r'/test_case/inter_case/' + file_name,mode='r',encoding='utf-8') as f:\n"}, "start": 1652772923610, "stop": 1652772923611}, {"name": "get_token", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\python_code\\\\pytest_demo/test_case/inter_case/extract.yaml'\n", "trace": "  File \"E:\\programs\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\programs\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\programs\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"E:\\python_code\\pytest_demo\\test_case\\inter_case\\conftest.py\", line 19, in get_token\n    return yam.read_inter_yaml('extract.yaml')['accessToken']\n  File \"E:\\python_code\\pytest_demo\\common\\yaml_util.py\", line 31, in read_inter_yaml\n    with open(self.root_path + r'/test_case/inter_case/' + file_name,mode='r',encoding='utf-8') as f:\n"}, "start": 1652772923645, "stop": 1652772923645}], "start": 1652772923645, "stop": 1652772923677}